pillowtop/listener.py

```diff
@@ -230,11 +207,11 @@ def change_trigger(self, changes_dict):
         if self.use_locking:
             lock = self.document_class.get_obj_lock_by_id(id)
             lock.acquire()
-            return LockManager(self.couch_db.open_doc(id), lock)
+            return LockManager(self._couch_db.open_doc(id), lock)
         elif changes_dict.get('doc', None) is not None:
             return changes_dict['doc']
         else:
-            return self.couch_db.open_doc(id)
+            return self._couch_db.open_doc(id)
```

Comment:
because i want to eventually `soft_assert` that nothing uses the public version and everything funnels through a `DocumentStore` object